internal/appinfo/runtime_info.go

```diff
@@ -157,7 +159,7 @@ func (h *RuntimeInfoHandler) planNameOrDefault(inst internal.InstanceWithOperati
 	if inst.ServicePlanName != "" {
 		return inst.ServicePlanName
 	}
-	return broker.Plans(h.plansConfig, "", false, false)[inst.ServicePlanID].Name
+	return broker.Plans(h.plansConfig, "", false, false, h.regionParameterIsRequired)[inst.ServicePlanID].Name //tu
```

Comment:
you are taking only a name here, the flag which indicates if parameter is required is not necessary here

Reply:
You're right, I didn't see that we only take the name parameter.